plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -151,7 +161,7 @@ public HiveColumnHandle toHiveColumnHandle()
     {
         return new HiveColumnHandle(
                 physicalName, // this name is used for accessing Parquet files, so it should be physical name
-                0, // hiveColumnIndex; we provide fake value because we always find columns by name
+                fieldId.orElse(0), // hiveColumnIndex; we provide fake value because we always find columns by name
```

Comment:
I think with how you have this now you can undo this change and leave it as 0 all the time, right?